packages/rs-drive-abci/src/execution/validation/state_transition/state_transitions/identity_credit_transfer/structure/v0/mod.rs

```diff
@@ -8,7 +8,7 @@ use dpp::state_transition::identity_credit_transfer_transition::accessors::Ident
 use dpp::state_transition::identity_credit_transfer_transition::IdentityCreditTransferTransition;
 use dpp::validation::SimpleConsensusValidationResult;
 
-const MIN_TRANSFER_AMOUNT: u64 = 1000;
+const MIN_TRANSFER_AMOUNT: u64 = 100000;
```

Comment:
Why so?. Please express this change in PR description

Reply:
I mean that was 1 duff before, even 100 duffs is very low.

Reply:
Could you please describe this change in the PR description (specifically in breaking change section)